### ISSUE TITLE
Renal cortical necrosis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5072]
Definition:Pale tissue with typical appearance of coagulative necrosis, affecting both tubules and glomeruli in the cortex. Features can include pyknotic nuclei, Simplified, flattened epithelium of proximal tubules. The tubular and glomerular basement membranes are still visible without viable nuclei 
, ref:https://www.ajkd.org/article/S0272-6386(16)00222-5/fulltext

**Comment by pnrobinson** [comment]
Definition: Patchy or diffuse ischemic destruction of all the elements of renal cortex resulting from significantly diminished renal arterial perfusion. Coagulative necrosis may be present, involving all tubular segments and glomeruli. Nuclei may be pale and pyknotic, or may no longer be apparent. Thrombi may be present in vessels at the edge of the infarct. PMID:26558184, PMID:27091021 

Comment: Cortical necrosis commonly manifests clinically as a rapid loss of glomerular filtration rate, often with oliguria, granular casts, and low-level proteinuria. The key diagnostic features of renal cortical necrosis are (i) pale tissue with typical appearance of coagulative
necrosis, affecting both tubules and glomeruli; (ii) pyknotic nuclei; and (iii) simplified, flattened epithelium of proximal tubules

**Comment by pnrobinson** [comment]
@yongqunh  This term exists already at https://hpo.jax.org/app/browse/term/HP:0025418
I will update the definition as shown above, OK?

**Comment by nicolevasilevsky** [comment]
We could add this logical def:

'has part' some 
    ('having extra processual parts'
     and ('inheres in part of' some 'cortex of kidney')
     and (towards some 'necrotic cell death')
     and ('has modifier' some abnormal))

Note- we don't have a necrosis pattern in uPheno. @matentzn should we add one?

**Comment by nicolevasilevsky** [comment]
Or I wonder if we could use this pattern:
https://github.com/obophenotype/upheno/blob/master/src/patterns/dosdp-dev/abnormallyIncreasedOccurrenceOfBiologicalProcessInLocation.yaml

but this is using the PATO term 'increased occurrence' instead of 'having extra processual parts'

**Comment by yongqunh** [comment]
The new definition appears good.

**Comment by pnrobinson** [comment]
@nicolevasilevsky  -- I have updated the English-language definition as above. Please go ahead with the logical definition. There are other classes that have the first kind of definition ('having extra processual parts'). Please close when done!

**Comment by nicolevasilevsky** [comment]
@matentzn pointed out there is a new necrosis pattern in uPheno:
https://github.com/obophenotype/upheno/blob/master/src/patterns/dosdp-dev/necroticAnatomicalEntity.yaml

'has_part' some ('necrotic' and ('inheres_in' some [anatomical entity]) and ('has_modifier' some 'abnormal'))"

@pnrobinson any objections to using this pattern through HPO instead?

I already added the PATO necrotic term to HPO

**Comment by pnrobinson** [comment]
@nicolevasilevsky  The above simplified pattern seems fine to me!
